state/logs.go

```diff
@@ -822,17 +833,35 @@ func PruneLogs(st ControllerSessioner, minLogTime time.Time, maxLogsMB int) erro
 			"t": bson.M{"$lt": thresholdTs},
 		})
 		if err != nil {
-			return errors.Annotate(err, "log pruning failed")
+			return "", errors.Annotate(err, "log pruning failed")
 		}
 		pruneCounts[modelUUID] += removeInfo.Removed
 	}
 
+	totalRemoved := 0
+	modelCount := 0
 	for modelUUID, count := range pruneCounts {
 		if count > 0 {
+			totalRemoved += count
+			modelCount++
 			logger.Debugf("pruned %d logs for model %s", count, modelUUID)
 		}
 	}
-	return nil
+
+	var removed string
+	if totalRemoved == 0 {
+		removed = "no pruning necessary"
+	} else {
+		s := "s"
```

Comment:
Hmm, maybe you are thinking of "golang.org/x/text/feature/plural"? 
I could not find any common utility that we use through our codebase. I agree that we should have something common since we require it in a few places and each place crafts plurality in strings all over again :)